plugins/processors/script/README.md

```diff
@@ -0,0 +1,252 @@
+# script
+
+`script` processor will run the user specified script to transform each asset
+that is emitted by the extractor. Currently, [Tengo][tengo] is the only
+supported script engine.
+
+Refer Tengo documentation for script language syntax and supported functionality
+\- https://github.com/d5/tengo/tree/v2.13.0#references
+. [Tengo standard library modules][tengo-stdlib] can also be imported and used
+if requried.
+
+## Usage
+
+```yaml
+processors:
+  - name: script
+    config:
+      engine: tengo
+      script: |
+        asset.owners = append(asset.owners || [], { name: "Big Mom", email: "big.mom@wholecakeisland.com" })
```

Comment:
should we also put more examples like how to merge map and loop through list? wdyt?

Reply:
I have added a thorough example already - https://github.com/odpf/meteor/blob/script-processor/plugins/processors/script/README.md#worked-example

Do you think this suffices or we should add a section showing script code for merging map, looping through list etc.?

Reply:
oh missed that part. lgtm then